services/galley/test/integration/API/Roles.hs

```diff
@@ -182,7 +182,7 @@ wireAdminChecks cid admin otherAdmin mem = do
   let activatedAccess = ConversationAccessUpdate [InviteAccess] NonActivatedAccessRole
   putAccessUpdate admin cid activatedAccess !!! assertActionSucceeded
   -- Update your own member state
-  let memUpdate = memberUpdate {mupOtrMute = Just True}
+  let memUpdate = memberUpdate {mupOtrArchive = Just True}
```

Comment:
Can you clarify this change? How come you use the archive attribute here instead of the muted status? The same question goes for the change on line 233 below.

Reply:
This is just testing that updating works. It doesn't really matter what we update specifically. Since `mupOtrMute` has been removed, I just randomly changed it to `mupOtrArchive`.